actionpack/test/controller/rate_limiting_test.rb

```diff
@@ -66,8 +66,9 @@ class RateLimitingTest < ActionController::TestCase
     get :limited
     assert_response :ok
 
-    get :limited
-    assert_response :too_many_requests
+    assert_raises ActionController::TooManyRequests do
+      get :limited
+    end
```

Comment:
This change won't have end-user facing consequences in production environments, but has the opportunity to impact consumer application test suites.

While this type of test change is necessary for the Action Dispatch test suite, they will not be necessary for applications that set `config.action_dispatch.show_exceptions = :all` (which is the default value).

However, test suites for applications that set `config.action_dispatch.show_exceptions = :none` will start to fail. Does that constitute enough of a breaking change to demand a new Railtie configuration and default?